### PR TITLE
Fix docstring format

### PR DESCRIPTION
**Pull request by triple-Mu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.
Describe the feature
As shown in the following picture, there are unpleasant blanks in the strings.
We need to remove them.
## Motivation

 There are unpleasant blanks in the strings in doc common，we need to remove them.
So I fix doc common and remove '\' which is used for switching a new line.

## Modification

Files change
`mmdeploy/apis/calibration.py ` , 
`mmdeploy/apis/inference.py` , 
`mmdeploy/apis/pytorch2onnx.py` , 
`mmdeploy/apis/visualize.py` , 
`mmdeploy/backend/tensorrt/onnx2tensorrt.py`

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by lvhan028** [comment]
Hi, @triple-Mu Could you please describe this PR more clearly in the title?

File: mmdeploy/apis/calibration.py [hunk 1]
```diff
@@ -24,14 +24,14 @@ def create_calib_table(calib_file: str,
...
         >>> calib_file = get_calib_filename(deploy_cfg)
-        >>> model_cfg = 'mmdetection/configs/fcos/' \
-            'fcos_r50_caffe_fpn_gn-head_1x_coco.py'
-        >>> model_checkpoint = 'checkpoints/' \
-            'fcos_r50_caffe_fpn_gn-head_1x_coco-821213aa.pth'
+        >>> model_cfg = ('mmdetection/configs/fcos/'
+                         'fcos_r50_caffe_fpn_gn-head_1x_coco.py')
+        >>> model_checkpoint = ('checkpoints/'
+                                'fcos_r50_caffe_fpn_gn-head_1x_coco-821213aa.pth')
         >>> create_calib_table(calib_file, deploy_cfg, \
```
Comment (lvhan028) [inline mmdeploy/apis/calibration.py:35]: Can '\\' be also removed from `create_calib_table`?

File: mmdeploy/apis/inference.py [hunk 2]
```diff
@@ -16,15 +16,15 @@ def inference_model(model_cfg: Union[str, mmcv.Config],
...
-        >>> deploy_cfg = 'configs/mmdet/detection/' \
-            'detection_onnxruntime_dynamic.py'
+        >>> model_cfg = ('mmdetection/configs/fcos/'
+                         'fcos_r50_caffe_fpn_gn-head_1x_coco.py')
+        >>> deploy_cfg = ('configs/mmdet/detection/'
+                          'detection_onnxruntime_dynamic.py')
         >>> backend_files = ['work_dir/fcos.onnx']
         >>> img = 'demo.jpg'
         >>> device = 'cpu'
         >>> model_output = inference_model(model_cfg, deploy_cfg, \
```
Comment (lvhan028) [inline mmdeploy/apis/inference.py:26]: Can '\\' be also removed from `inference_model`?

File: mmdeploy/backend/tensorrt/onnx2tensorrt.py [hunk 3]
```diff
@@ -26,8 +26,8 @@ def onnx2tensorrt(work_dir: str,
         >>> work_dir = 'work_dir'
         >>> save_file = 'end2end.engine'
         >>> model_id = 0
-        >>> deploy_cfg = 'configs/mmdet/detection/' \
-                'detection_tensorrt_dynamic-320x320-1344x1344.py'
+        >>> deploy_cfg = ('configs/mmdet/detection/'
+                          'detection_tensorrt_dynamic-320x320-1344x1344.py')
         >>> onnx_model = 'work_dir/end2end.onnx'
         >>> onnx2tensorrt(work_dir, save_file, model_id, deploy_cfg, \
```
Comment (lvhan028) [inline mmdeploy/backend/tensorrt/onnx2tensorrt.py:32]: Can '\\' be also removed from `onnx2tensorrt`?

**Review by lvhan028 — APPROVED** [review]

**Comment by lvhan028** [comment]
related issue #495

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**